libs/libs/integrations/twilio.star

```diff
@@ -0,0 +1,65 @@
+_BASE_URL = "https://api.twilio.com/2010-04-01"
+
+def connect(conn, account_sid, auth_token=None):
+    """Connect to the Twilio API"""
+
+    def _url(path):
+        return _BASE_URL + path
+
+    def _post(path, body):
+        resp = conn.post(
+            _url(path), 
+            form_body=body, 
+            basic_auth=auth_token and (account_sid, auth_token) or None,
+        )
+        
+        json = resp.body.json()
+
+        if resp.status_code < 200 or resp.status_code >= 400:
+            fail(json)
+            
+        return json
```

Comment:
This part is probably shared utils for all http clients, can be lib/integrations/http
usable by itself 
small effort to split it now and we can get a lot by reusing it